### PR TITLE
feat(unblock): Add support for the unblock SUMO link.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
@vladikoff - r?

With this PR, the link appears in the unblock screen:

screen shot 2016-11-15 at 12 55 08

Fixes https://github.com/mozilla/fxa-content-server/issues/4288

### CODE REVIEW

File: account-blocked-sumo-link/account-blocked-sumo-link.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+module.exports = {
+  name: 'The account blocked SUMO link',
+  startDate: '2015-01-01',
+  independentVariables: ['blockedSigninSupportUrl'],
+  eligibilityFunction: function (subject) {
+    return true;
+  },
+  groupingFunction: function (subject) {
+    return {
+      blockedSigninSupportUrl: 'https://support.mozilla.org/kb/accounts-blocked'
```
Comment (shane-tomlinson) [inline account-blocked-sumo-link/account-blocked-sumo-link.js:12]: I asked Joni from SUMO whether we need to add any utm_\* parameters to the link.

**Comment by vladikoff** [comment]
codepen to show inconsistency in browsers: https://codepen.io/vladikoff/pen/RoGbOG?editors=1111

Chrome reports width 199, FF reports 174

**Comment by vladikoff** [comment]
@shane-tomlinson we need to fix up the css for the anchors in some way. This bug is caused by a diff in browser breaking on different characters. firefox breaks hte line on `/` by default and Chrome on `-`

File: account-blocked-sumo-link/account-blocked-sumo-link.js (hunk 1 above)
Comment (shane-tomlinson) [reply account-blocked-sumo-link/account-blocked-sumo-link.js:12]: The response in https://bugzilla.mozilla.org/show_bug.cgi?id=1310822#c16 is:

> In that case, you can link to an article from a website without the version/OS/Locale/ values. You can link directly to: https://support.mozilla.org/kb/accounts-blocked

**Comment by shane-tomlinson** [comment]
The SUMO link is only converted to text for Fennec and Fx for iOS:

```bash
➜  auth_brokers git:(master) ✗ rgrep -i external .                   git:(master|…
./base.js:       * Should external links be converted to text?
./base.js:      convertExternalLinksToText: false,
./fx-fennec-v1.js:      convertExternalLinksToText: true,
./fx-ios-v1.js:      convertExternalLinksToText: true
```

If it looks good on these two I think we should be good to go. Building Fx for iOS now to see.

**Comment by vladikoff** [comment]
Yea I tested this on Webkit and it was fine. I guess we can merge it with the notion that "Blink users should never see this". Feel free to merge when you are around so it shows up in prod 👍

**Comment by shane-tomlinson** [comment]
Fx for iOS:

screen shot 2016-11-24 at 11 41 55

Fennec:

screenshot_20161124-114421

**Pull request merged**